src/__tests__/sms/fixtures.ts

```diff
@@ -337,4 +337,84 @@ export const fixtures = {
     BIND(template("http://api.stardog.com/sms_bind/template={id}") AS ?template)
   }
   `,
+  // Taken from Stardog docs: https://docs.stardog.com/virtual-graphs/mapping-data-sources#edge-properties-in-sms2
+  edgePropertiesEmbeddedTriples: `
+  PREFIX emp: <http://example.com/emp>
+
+  MAPPING
+  FROM SQL {
+    select * from employees
+  }
+  TO {
+    ?emp a emp:Employee .
+
+    <<?emp emp:firstName ?first_name>> emp:since "?birth_date"^^xsd:date .
+  }
+  WHERE {
+    BIND(template("http://employee/{emp_no}") as ?emp)
+  }`,
+  // Taken from Stardog:
+  edgePropertiesEmbeddedTriples2: `
+  # Employees mappings with edge properties
+
+  PREFIX emp: <http://example.com/emp/>
+
+  MAPPING
+  FROM SQL {
+    select * from employees
+  }
+  TO {
+    ?emp a emp:Employee .
+
+    # TODO: Should we avoid duplication of implicit ordinary scans?
+    <<?emp emp:firstName ?first_name>> emp:since "?birth_date"^^xsd:date .
+
+    <<?emp emp:firstName ?first_name>> emp:justAnotherFirstNameEdgeProp "?hire_date"^^xsd:date .
+
+    <<?emp emp:firstName ?first_name>> emp:beginsAt "?birth_date"^^xsd:date .
+    <<?emp emp:lastName ?last_name>> emp:beginsAt "?birth_date"^^xsd:date .
+    <<?emp emp:worksFor emp:TheCompany>> emp:beginsAt "?hire_date"^^xsd:date .
+  }
+  WHERE {
+    BIND(template("http://employee/{emp_no}") as ?emp)
+  }`,
+  edgePropertiesEmbeddedPropertyList: `
+  PREFIX emp: <http://example.com/emp>
+
+  MAPPING
+  FROM SQL {
+    select * from employees
+  }
+  TO {
+    ?emp a { :since 2010 } emp:Employee .
+  }
+  WHERE {
+    BIND(template("http://employee/{emp_no}") as ?emp)
+  }
+  `,
+  // Taken from Stardog:
+  edgePropertiesEmbeddedPropertyList2: `
+  # Employees mappings with edge properties using Stardog syntax
+
+  PREFIX emp: <http://example.com/emp/>
+
+  MAPPING
+  FROM SQL {
+    select * from employees
+  }
+  TO {
+    ?emp a emp:Employee .
+
+    ?emp emp:firstName {
+      emp:since "?birth_date"^^xsd:date ;
+      emp:justAnotherFirstNameEdgeProp "?hire_date"^^xsd:date ;
+      emp:beginsAt "?birth_date"^^xsd:date
+    } ?first_name .
```

Comment:
Maybe I'm doing something wrong, but I'm unable to save anything like this to a VG right now (I can save random edge properties, though).

Reply:
I just checked and was able to stuff like this without an issue. Maybe it was specific to what you were doing?

In any case, this is valid syntax (whether it is currently working when submitted) and is a test case used in Stardog's own tests. So, given that it works for me and _ought_ to work either way, I don't think it should block merging here. Maybe we can follow-up on your specific case to figure out what the issue is.